### PR TITLE
Fixing the import error and KeyError in snappi_test/multidut executions.

### PR DESCRIPTION
**Pull request by rraghav-cisco**
@sdszhang : This PR attempts to fix the fixture-not-found error and KeyError that you are seeing in the snappi_test multidut runs. Pls let me know if this works for you.

### CODE REVIEW

File: tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py [hunk 1]
```diff
@@ -172,8 +172,10 @@ def test_pfcwd_basic_multi_lossless_prio_reboot(snappi_api,                 # no
                                                 localhost,
                                                 duthosts,
                                                 enum_dut_lossless_prio_with_completeness_level,   # noqa: F811
```
Comment (sdszhang) [inline tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py:174]: @rraghav-cisco   'duthosts' and 'enum_dut_lossless_prio_with_completeness_level' are not used for this test case anymore. pls remove these two.

Comment (rraghav-cisco) [reply tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py]: Done.

**Comment by rraghav-cisco** [comment]
There are some tests that are flaky, they passed in one round, and failed in second round.
testsuite name="pytest" errors="0" failures="3" skipped="16" tests="40" time="20466.641"

File: tests/snappi_tests/files/helper.py [hunk 2]
```diff
@@ -131,11 +133,15 @@ def save_config_and_reboot(node, results=None):
...
         logger.info("Wait until the system is stable")
         wait_until(180, 20, 0, node.critical_services_fully_started)
+        wait_until(180, 20, 0, check_interface_status_of_up_ports, node)
 
     # Convert the list of duthosts into a list of tuples as required for parallel func.
     args = set((snappi_ports[0]['duthost'], snappi_ports[1]['duthost']))
     parallel_run(save_config_and_reboot, {}, {}, list(args), timeout=900)
 
+    # Wait for internal bgp to come up.
+    time.sleep(100)
```
Comment (sdszhang) [inline tests/snappi_tests/files/helper.py:143]: same comment to add bgp check instead of hard coded value.

Comment (rraghav-cisco) [reply tests/snappi_tests/files/helper.py]: @sdszhang : Done.

**Comment by yejianquan** [comment]
@rraghav-cisco , please paste test result so that I can merge it, thanks

**Comment by rraghav-cisco** [comment]
Test results, seeing flaky test fails, but this PR is not the problem:

```
================================================================================================================== short test summary info ===================================================================================================================
PASSED snappi_tests/multidut/pfc/test_lossless_response_to_external_pause_storms.py::test_lossless_response_to_external_pause_storms_test[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_lossless_response_to_external_pause_storms.py::test_lossless_response_to_external_pause_storms_test[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_lossless_response_to_throttling_pause_storms.py::test_lossless_response_to_throttling_pause_storms[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_lossless_response_to_throttling_pause_storms.py::test_lossless_response_to_throttling_pause_storms[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_m2o_fluctuating_lossless.py::test_m2o_fluctuating_lossless[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_m2o_fluctuating_lossless.py::test_m2o_fluctuating_lossless[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossless.py::test_m2o_oversubscribe_lossless[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossless.py::test_m2o_oversubscribe_lossless[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossless_lossy.py::test_m2o_oversubscribe_lossless_lossy[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossless_lossy.py::test_m2o_oversubscribe_lossless_lossy[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossy.py::test_m2o_oversubscribe_lossy[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossy.py::test_m2o_oversubscribe_lossy[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info0-yy39top-lc4|3]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info0-yy39top-lc4|4]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info1-yy39top-lc4|3]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info1-yy39top-lc4|4]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_multi_lossless_prio[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_multi_lossless_prio[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio_reboot[multidut_port_info0-cold-yy39top-lc4|4]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio_reboot[multidut_port_info1-cold-yy39top-lc4|4]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_multi_lossless_prio_reboot[multidut_port_info0-cold]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_multi_lossless_prio_reboot[multidut_port_info1-cold]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|1]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|2]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|5]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|6]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|1]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|2]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|5]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|6]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio[multidut_port_info0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio[multidut_port_info1]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio_reboot[multidut_port_info0-cold-yy39top-lc4|0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio_reboot[multidut_port_info1-cold-yy39top-lc4|0]
PASSED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio_reboot[multidut_port_info1-cold]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio[multidut_port_info0-yy39top-lc4|4-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio[multidut_port_info0-yy39top-lc4|4-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio[multidut_port_info1-yy39top-lc4|4-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio[multidut_port_info1-yy39top-lc4|4-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio[multidut_port_info0-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio[multidut_port_info0-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio[multidut_port_info1-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio[multidut_port_info1-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_reboot[multidut_port_info0-cold-yy39top-lc4|4-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_reboot[multidut_port_info0-cold-yy39top-lc4|4-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_reboot[multidut_port_info1-cold-yy39top-lc4|4-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_reboot[multidut_port_info1-cold-yy39top-lc4|4-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_reboot[multidut_port_info0-cold-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_reboot[multidut_port_info0-cold-False]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_reboot[multidut_port_info1-cold-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_service_restart[multidut_port_info0-yy39top-lc4|4-True-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_service_restart[multidut_port_info0-yy39top-lc4|4-False-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_service_restart[multidut_port_info1-yy39top-lc4|4-True-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio_service_restart[multidut_port_info1-yy39top-lc4|4-False-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_restart_service[multidut_port_info0-True-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_restart_service[multidut_port_info0-False-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_restart_service[multidut_port_info1-True-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_restart_service[multidut_port_info1-False-swss]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_burst_storm_with_snappi.py::test_pfcwd_burst_storm_single_lossless_prio[multidut_port_info0]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_burst_storm_with_snappi.py::test_pfcwd_burst_storm_single_lossless_prio[multidut_port_info1]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_m2o_with_snappi.py::test_pfcwd_many_to_one[multidut_port_info0-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_m2o_with_snappi.py::test_pfcwd_many_to_one[multidut_port_info1-True]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_runtime_traffic_with_snappi.py::test_pfcwd_runtime_traffic[multidut_port_info0]
PASSED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_runtime_traffic_with_snappi.py::test_pfcwd_runtime_traffic[multidut_port_info1]
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_global_pause_with_snappi.py:17: Global pause is not supported in cisco-8000.
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:139: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:139: Reboot type fast is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:199: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py:199: Reboot type fast is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:131: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:131: Reboot type fast is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:188: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [2] snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py:188: Reboot type fast is not supported on cisco-8000 switches
SKIPPED [4] snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py:143: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [4] snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py:143: Reboot type fast is not supported on cisco-8000 switches
SKIPPED [4] snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py:194: Reboot type warm is not supported on cisco-8000 switches
SKIPPED [4] snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py:194: Reboot type fast is not supported on cisco-8000 switches
ERROR snappi_tests/multidut/pfc/test_lossless_response_to_external_pause_storms.py::test_lossless_response_to_external_pause_storms_test[multidut_port_info1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_lossless_response_to_throttling_pause_storms.py::test_lossless_response_to_throttling_pause_storms[multidut_port_info1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_m2o_oversubscribe_lossless_lossy.py::test_m2o_oversubscribe_lossless_lossy[multidut_port_info1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info1-yy39top-lc4|3] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_single_lossless_prio[multidut_port_info1-yy39top-lc4|4] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py::test_pfc_pause_multi_lossless_prio[multidut_port_info0] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|2] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info0-yy39top-lc4|6] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|0] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|2] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|5] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_single_lossy_prio[multidut_port_info1-yy39top-lc4|6] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio[multidut_port_info0] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio[multidut_port_info1] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_single_lossless_prio[multidut_port_info1-yy39top-lc4|4-False] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
ERROR snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio[multidut_port_info0-False] - Failed: Processes "['analyze_logs--<MultiAsicSonicHost yy39top-lc4>']" failed with exit code "1"
FAILED snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py::test_pfc_pause_multi_lossy_prio_reboot[multidut_port_info0-cold] - Failed: Background Flow Prio 3 should not have any dropped packet
FAILED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_a2a_with_snappi.py::test_multidut_pfcwd_all_to_all[multidut_port_info0-False] - Failed: Test Flow 1 -> 0 Prio 4 should not have any dropped packet
FAILED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_a2a_with_snappi.py::test_multidut_pfcwd_all_to_all[multidut_port_info1-False] - Failed: Test Flow 1 -> 0 Prio 3 should not have any dropped packet
FAILED snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py::test_pfcwd_basic_multi_lossless_prio_reboot[multidut_port_info1-cold-False] - Failed: Loss rate of Data Flow 1 (1.0) should be in [0, 0]
====================================================================================== 4 failed, 66 passed, 34 skipped, 88 warnings, 18 errors in 42397.96s (11:46:37) =======================================================================================
sonic@ixia-sonic-mgmt-whitebox:/data/tests$
```

**Pull request merged**